CONTRIBUTING.md

```diff
@@ -56,6 +81,20 @@ To run tests:
 $ composer test
 ```
 
+### RUNNING UNIT TESTS ON DOCKER
+
+You may run the unit tests through the container without running bash via
+
+```
+docker-composer run --rm php
```

Comment:
I removed the command form `Dockerfile` so it is no longer working. In previous section we already have how to run tests. We can note that is it possible also tu run tests or just command using:
```console
$ docker-compose run --rm php composer test
```